src/authentication/templates/api/models/User.js

```diff
@@ -56,6 +56,7 @@ export default {
   },
 
   beforeUpdate(values, next) {
+    if (false === values.hasOwnProperty('password')) return next()
```

Comment:
Mainly for readability more clear what we are checking against and putting it on the left side avoids accidentally using =